job/server/src/main/java/alluxio/master/AlluxioJobMasterProcess.java

```diff
@@ -84,6 +84,9 @@
     mFileSystem = FileSystem.Factory.create(mFsContext);
     mUfsManager = new JobUfsManager();
     try {
+      if (!mJournalSystem.isFormatted()) {
```

Comment:
How about below?

```
if (!mJournalSystem.isFormatted()) {
        throw new RuntimeException(
            String.format("Journal %s has not been formatted!", mJournalSystem));
      }
```

This is what we do for master processes.

Reply:
Before commit, I see the master processes. There is two reasons not using it.

1.  Not support to format the job master journal in the shell `https://github.com/Alluxio/alluxio/blob/master/bin/alluxio`
2. For users, it's friendly to avoid of manual format.

Reply:
I think it's OK for the job-master processes.